src/Components/Blazor/Build/testassets/razorclasslibrary/RazorClassLibrary.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <ReferenceBlazorBuildLocally>true</ReferenceBlazorBuildLocally>
+    <RazorLangVersion>3.0</RazorLangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Components" Version="3.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="3.0.0" />
```

Comment:
Could we remove this reference?

Reply:
This is straight out of the template. I imagine we would want to do the change there. (and here potentially)

Reply:
Removed